model/variable.cc

```diff
@@ -459,10 +459,17 @@ void VariableBase::adjustSliderBounds() const
       {
         if (sliderMax<vv->value()) sliderMax=vv->value();
         if (sliderMin>vv->value()) sliderMin=vv->value();
-        if ((sliderMax-sliderMin)/sliderStep > 1.0e04) sliderStep=(sliderMax-sliderMin)/1.0e04;   // ensure there are at most 10000 steps between sliderMin and Max. for ticket 1255.     
+        sliderStep=maxSliderSteps();   
       }
 }
 
+double VariableBase::maxSliderSteps() const
+{
+    // ensure there are at most 10000 steps between sliderMin and Max. for ticket 1255. 	
+	if ((sliderMax-sliderMin)/sliderStep > 1.0e04) sliderStep=(sliderMax-sliderMin)/1.0e04;    
```

Comment:
Kind of OK, but this method does lie about the fact that it mutates sliderStep. Better would be to return (sliderMax-sliderMin)/1.0e04 at this point, rather than setting it. Then it would truly live up to the const attribute. FWIW, I can't remember now why these attributes are mutable.